guides/common/assembly_updating-server-smart-proxy-hosts.adoc

```diff
@@ -0,0 +1,17 @@
+ifdef::context[:parent-context: {context}]
+
+[id="Updating_Server_SmartProxy_Content_Hosts_{context}"]
+= Updating {ProjectServer}, {SmartProxyServerTitle}, and Content Hosts
```

Comment:
I see content hosts listed here in the title, paragraph below and the prerequisites for `Updating Satellite Server`, but I don't see a section for them. Is the process the same as updating capsule? If so we should also add content hosts to the `Updating Capsule Server` section or if not they probably need their own section.

Reply:
Agree with your view. I see the content in the old guide. Either we need to have a whole new section on `Content Host` or we need to drop it completely. There is no section or mentioning of `Updating Content Host` in the old guide.